wellregistry/registry/tests/admin/test_auto_complete.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tests for admin.monitoring_location for Autocomplete module
+"""
+from django.contrib.admin.sites import AdminSite
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.test import Client, TestCase
+from ...admin.monitoring_location import MonitoringLocationAdmin
+from ...models import MonitoringLocation
+class TestAutoCompleteView(TestCase):
+    # pylint: disable=too-many-instance-attributes
+    fixtures = ['test_groups.json', 'test_altitude_datum.json', 'test_counties.json',
+                'test_countries.json', 'test_horizontal_datum.json', 'test_nat_aquifer.json',
+                'test_states.json', 'test_units.json', 'test_user.json', 'test_agencies.json',
+                'test_monitoring_location.json']
+
+    def setUp(self):
+        self.superuser = get_user_model().objects.create_superuser('my_superuser')
```

Comment:
You do not need all of these users for this test. Just create the self.usgs_user

Reply:
fixed